decidim-meetings/app/views/decidim/meetings/registration_mailer/confirmation.html.erb

```diff
@@ -6,5 +6,5 @@
 <% if @meeting.customize_registration_email? %>
   <p><%= present(@meeting).registration_email_custom_content %></p>
 <% end %>
-
+<br>
```

Comment:
After discussing with @alecslupu, we agreed to use the `<br>` tag to style the custom registration email content as it suits all browsers and platforms.

Reply:
Yeah, HTML/CSS in emails is like programming in the year 2003